core/src/test/java/org/jdbi/v3/core/qualifier/TestNVarchar.java

```diff
@@ -76,18 +76,18 @@ public void sqlStatementBindNVarchar() {
 
             assertThat(
                 handle.select("SELECT name FROM nvarchars ORDER BY id")
-                    .mapTo(String.class, NVarchar.class)
+                    .mapTo(QualifiedType.of(String.class).with(NVarchar.class))
                     .list())
                 .containsExactly("foo", "bar", "baz", "qux");
 
             assertThat(
                 handle.select("SELECT name FROM nvarchars ORDER BY id")
-                    .mapTo(new GenericType<String>() {}, NVarchar.class)
+                    .mapTo(QualifiedType.of(new GenericType<String>() {}).with(NVarchar.class))
                     .list())
                 .containsExactly("foo", "bar", "baz", "qux");
 
             List rawList = handle.select("SELECT name FROM nvarchars ORDER BY id")
-                .mapTo(NVARCHAR_STRING)
+                .mapTo((QualifiedType<?>) NVARCHAR_STRING)
```

Comment:
Why is this cast necessary?  That seems like a real wart that you need to cast it to remove the generic type?

Reply:
It's not necessary at all, it's purely to help demonstrate the point of that test, to neglect the generic type.